tests/test_user_management.py

```diff
@@ -25,7 +31,27 @@ def test_success(self):
 
         self.assertCountEqual(actual.keys(), expected_keys)
 
-    def test_failure(self):
+    def test_user_by_token_failure(self):
         with self.assertRaises(requests.exceptions.HTTPError):
             _ = get_user_information('https://cloud-backend.lanalabs.com',
                                      token='not-a-valid-token ')
+
+    def test_list_users(self):
+        users = self.api.get_all_users()
+        self.assertGreaterEqual(len(users), 1)
+
+    def test_user_management(self):
+        resp_user_info = self.api.get_user_information_by_id('e44e8b55-1b28-4db7-9396-6acd4e94ee2a')
```

Comment:
This test also relies on a source system state we are unable to reproduce. What we could do is using the api user. When creating an api instance, it stores the user information under `self.api.user`. So we could require credentials for two roles in the same organisation for the test. One needs to be at least suer admin, the other e.g. ranalyst. We can then write

```python
class TestUsersAPI(unittest.TestCase):

    @classmethod
    def setUpClass(cls) -> None:
        with open('./tests/admin_config.json') as f:
            cls.credentials_user_admin = json.load(f)
        cls.api_user_admin = create_api(verify=True, **cls.credentials_user_admin)

        with open('./tests/admin_config.json') as f:
            cls.credentials_ranalyst = json.load(f)
        cls.api_r_analyst = create_api(verify=True, **cls.credentials_ranalyst)

    def test_user_management(self):
        resp_user_info = self.api_user_admin.get_user_information_by_id(self.api_r_analyst.user.user_id)
```

This way the test only relies on the state of LANA as specified in the credential jsons.